src/crunch.py

```diff
@@ -152,10 +152,8 @@ def main(argv):
         optimize_png(png_path_list[0])
         sys.exit(0)
     else:
-        processes = PROCESSES
         # if not defined by user, start by defining spawned processes as number of available cores
-        if processes == 0:
-            processes = cpu_count()
+        processes = PROCESSES or cpu_count()
```

Comment:
Can I ask you to revert these changes with the definition of `processes` back to the prior format?  I know that it is more verbose and perhaps not ideal style but it is more explicit and clear what it is doing for maintenance down the road when I forget all of this code... :)